src/current/v23.1/molt-verify.md

```diff
@@ -37,12 +37,12 @@ The following databases are currently supported:
   - For Linux: `molt.linux.amd64`
 
     Rename the binary to `molt` and add it to your `PATH` so you can execute the `molt verify` command from any shell.
+
 1. Get the connection strings for the source database and [CockroachDB](connect-to-the-database.html).
 1. Make sure the SQL user running MOLT Verify has read privileges on the necessary tables.
-   
 1. Run MOLT Verify: 
 
-    The `molt verify` command takes two or more SQL connection strings as arguments. You can append a name for easier readability using `<name>===` in front of the connection string. The first argument is considered the "source of truth". 
+    The `molt verify` command takes two or more SQL connection strings as arguments.
```

Comment:
not related to this PR exactly, but is it true that it can take more than 2 connection strings? I don't see how that works or why someone would do that if it did work.

Reply:
@otan ^

Reply:
missed this! it used to - it always compared results against the 1st one which acted as a source of truth. but then i decided it was annoying, so i removed that functionality. this should just read "two connections strings --source and --target as arguments`.